src/components/RichTextEditor/Plugins/UnderlinePlugin/index.ts

```diff
@@ -2,10 +2,21 @@
 
 import { createUnderlinePlugin } from '@udecode/plate';
 import { UNDERLINE_PLUGIN } from './id';
-import { UnderlineMarkupElement } from './UnderlineMarkupElement';
+import { UNDERLINE_CLASSES, UnderlineMarkupElement } from './UnderlineMarkupElement';
 import { UnderlineButton } from './UnderlineButton';
 import { Plugin, PluginProps } from '../Plugin';
 
+const PLATE_UNDERLINE_RULES = [
+    {
```

Comment:
Is this necessary if you overwrite it afterwards?

Reply:
And why dont you put everything in the `PLATE_UNDERLINE_RULES `?

Reply:
I added the const there to explicitly say which rules were coming from plate and not us, since I thought there should be some degree of separation between the rules so it's easier to tell which rule is important for the markup element